+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Tue May  9 00:13:33 2023 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL usbutils
+ pkgs=usbutils
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs usbutils --node 1 --tempfile ''
+ tmpfile2=/tmp/tmp3xh70ohw
+ '[' -z '' ']'
+ tmpfile=/tmp/tmp3xh70ohw
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Tue May  9 00:13:49 2023 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Tue May  9 00:13:50 2023 - INFO  - Start to run test.
+ lsusb
+ grep Bus
+ CHECK_RESULT 1 0 0 'error displaying USB device information'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='error displaying USB device information'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'error displaying USB device information'
+ LOG_ERROR 'error displaying USB device information'
+ message='error displaying USB device information'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'error displaying USB device information'
Tue May  9 00:13:50 2023 - ERROR - error displaying USB device information
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lsusb.sh line 30'
+ message='oe_test_lsusb.sh line 30'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_lsusb.sh line 30'
Tue May  9 00:13:50 2023 - ERROR - oe_test_lsusb.sh line 30
+ '[' 0 -eq 1 ']'
+ return 0
+ lsusb -v
+ grep 'Device Descriptor'
+ CHECK_RESULT 1 0 0 'detailed display of USB device information'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='detailed display of USB device information'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'detailed display of USB device information'
+ LOG_ERROR 'detailed display of USB device information'
+ message='detailed display of USB device information'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'detailed display of USB device information'
Tue May  9 00:13:51 2023 - ERROR - detailed display of USB device information
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lsusb.sh line 32'
+ message='oe_test_lsusb.sh line 32'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_lsusb.sh line 32'
Tue May  9 00:13:51 2023 - ERROR - oe_test_lsusb.sh line 32
+ '[' 0 -eq 1 ']'
+ return 0
+ find /dev/bus/usb/001/001
find: ‘/dev/bus/usb/001/001’: No such file or directory
+ CHECK_RESULT 1 0 0 'the linked device does not exist'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='the linked device does not exist'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'the linked device does not exist'
+ LOG_ERROR 'the linked device does not exist'
+ message='the linked device does not exist'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'the linked device does not exist'
Tue May  9 00:13:51 2023 - ERROR - the linked device does not exist
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lsusb.sh line 34'
+ message='oe_test_lsusb.sh line 34'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_lsusb.sh line 34'
Tue May  9 00:13:51 2023 - ERROR - oe_test_lsusb.sh line 34
+ '[' 0 -eq 1 ']'
+ return 0
+ lsusb -D /dev/bus/usb/001/001
+ grep 'Device Descriptor'
Cannot open /dev/bus/usb/001/001
+ CHECK_RESULT 1 0 0 'detailed display of USB device information'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='detailed display of USB device information'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'detailed display of USB device information'
+ LOG_ERROR 'detailed display of USB device information'
+ message='detailed display of USB device information'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'detailed display of USB device information'
Tue May  9 00:13:52 2023 - ERROR - detailed display of USB device information
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lsusb.sh line 36'
+ message='oe_test_lsusb.sh line 36'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_lsusb.sh line 36'
Tue May  9 00:13:52 2023 - ERROR - oe_test_lsusb.sh line 36
+ '[' 0 -eq 1 ']'
+ return 0
+ lsusb -t
+ grep /:
+ CHECK_RESULT 1 0 0 'display lsusb tree structure error'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='display lsusb tree structure error'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'display lsusb tree structure error'
+ LOG_ERROR 'display lsusb tree structure error'
+ message='display lsusb tree structure error'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'display lsusb tree structure error'
Tue May  9 00:13:53 2023 - ERROR - display lsusb tree structure error
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lsusb.sh line 38'
+ message='oe_test_lsusb.sh line 38'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_lsusb.sh line 38'
Tue May  9 00:13:53 2023 - ERROR - oe_test_lsusb.sh line 38
+ '[' 0 -eq 1 ']'
+ return 0
+ lsusb -V
+ grep usbutils
lsusb (usbutils) 015
+ CHECK_RESULT 0 0 0 'error displaying lsusb version information'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='error displaying lsusb version information'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Tue May  9 00:13:53 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 5
+ test 5 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue May  9 00:13:53 2023 - ERROR - failed to execute the case.
+ exit 5
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Tue May  9 00:13:54 2023 - INFO  - start environment cleanup.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmp3xh70ohw ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmp3xh70ohw
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Tue May  9 00:13:59 2023 - INFO  - Finish environment cleanup!
